atlabs/README.md

```diff
@@ -64,7 +65,7 @@ Your Africa's Talking issued virtual phone number.
 
 Phone number to dial
 
-```python3
+```python
```

Comment:
Did you know python and python3 are different, hence syntax highlighting will be different